src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -24,6 +30,7 @@ message NamespaceOptions {
     RetentionOptions retentionOptions = 6;
     bool snapshotEnabled              = 7;
     IndexOptions indexOptions         = 8;
+    repeated SchemaOption schemaOptions = 9;
```

Comment:
nit: mind adding a comment about the intent behind `repeated` here. Took me a second to figure out that you were retaining the history of all schemas. 

also, align on `=` within the struct please.

Reply:
I wonder if it might be better to move the `repeated` into the `SchemaOptions` itself? In the future we'd like to support multiple schemas per namespace so it might be nice if in the future we only need to modify the `SchemaOptions` struct instead of having to touch the `NamespaceOptions` struct too

Reply:
agree, will make the definition future proof

Reply:
> nit: mind adding a comment about the intent behind `repeated` here. Took me a second to figure out that you were retaining the history of all schemas.
> 
> also, align on `=` within the struct please.

regarding align =  in proto, do you have an easy way to do this format? I am intelliJ ide with proto plugin, it does not seem to do it for me. 

also, what's the rational for align =, golang format does not do that.